airflow/www_rbac/api/experimental/endpoints.py

```diff
@@ -168,18 +168,21 @@ def task_instance_info(dag_id, execution_date, task_id):
     """
     Returns a JSON with a task instance's public instance variables.
     The format for the exec_date is expected to be
-    "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15". This will
+    "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15" or
+    "YYYY-mm-DDTHH:MM:SS.f", for example: "2016-11-16T11:34:15.680399". This will
     of course need to have been encoded for URL in the request.
     """
 
     # Convert string datetime into actual datetime
     try:
-        execution_date = timezone.parse(execution_date)
```

Comment:
You are removing timezone information here and rely on your own parsing mechanism. I don't think that's a good idea?

Reply:
@bolkedebruin, oh shame! I don't pay attention to what it was changed from 1.9 and just apply my fix from 1.9, sorry for that (it was - https://github.com/apache/incubator-airflow/blob/v1-9-stable/airflow/www/api/experimental/endpoints.py#L126)

so, maybe this task not actually already in 1.10?

Reply:
yes, it works correct in 1.10, sorry for PR, I think need to close task - https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-1823